src/components/LoginPanel.tsx

```diff
@@ -54,10 +54,12 @@ const LoginPanel: React.FC<{}> = () => {
     const theme = useTheme();
 
     const [loaded, setLoaded] = React.useState(false);
+    const [baseUrl, setBaseUrl] = useState<string | null>(null);
 
     useEffect(() => {
+        setBaseUrl(window.location.origin);
         setLoaded(true);
-    }, []);
```

Comment:
is this intentional?

Reply:
I've put baseUrl in the dependency - not sure why it thinks I've removed the full line.